beater/request/context.go

```diff
@@ -126,7 +128,7 @@ func (c *Context) Reset(w http.ResponseWriter, r *http.Request) {
 
 // Header returns the http.Header of the context's writer
 func (c *Context) Header() http.Header {
-	return c.w.Header()
+	return c.W.Header()
 }
```

Comment:
I'd suggest that we either:
 - remove the method and change callers to `Context.W.Header()` (btw I'd also prefer to spell out the field name as "ResponseWriter", so it becomes `Context.ResponseWriter.Header()`), or
 - just embed `http.ResponseWriter` so methods are available on the `Context` type directly, and change callers to `Context.Header()`. Since Context holds both request & response, this is probably a bit unclear as to whether it refers to request or response headers.

Reply:
I went for the first suggestion in 5fdff5d11.